src/App.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './component/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC<{}> = () => {
+  const [goods, setGoods] = useState([]);
+
+  return (
+    <div className="box">
+      <h1 className="title">Dynamic list of Goods</h1>
+
+      <button
+        className="button is-success"
+        type="button"
+        onClick={async () => (
+          setGoods(await getAll()))}
```

Comment:
Will be better to store the result of async functions in some variable and use this variable in setGoods